dcos/tools.go

```diff
@@ -7,13 +7,31 @@ import (
 	"io/ioutil"
 	"net/http"
 	"os"
+	"strconv"
 	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
 	"github.com/sirupsen/logrus"
 
 	"github.com/dcos/dcos-diagnostics/util"
 )
 
+var (
+	findMasterNodesHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
+		Name: "find_master_node_time_seconds",
+		Help: "Time taken find single master node",
+	})
+)
+
+var (
+	doRequestHistogram = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name: "do_request_duration_seconds",
```

Comment:
Can we find a better name for this? `do_request` is really bound to the name of the method. What about something like `dcos_internal_request`?